src/main/java/teammates/ui/webapi/CreateCourseAction.java

```diff
@@ -43,10 +46,17 @@ public JsonResult execute() {
         String newCourseId = courseCreateRequest.getCourseId();
         String newCourseName = courseCreateRequest.getCourseName();
 
+        String institute = Const.UNKNOWN_INSTITUTION;
+        AccountAttributes account = logic.getAccount(userInfo.getId());
+        if (account != null && !StringHelper.isEmpty(account.getInstitute())) {
+            institute = account.getInstitute();
+        }
```

Comment:
Understand that this PR is only part of the bigger scheme of things, I believe `institute` will eventually be removed from the `Account` object?

Reply:
Yes. In general, schema-affecting changes have to be done in multiple steps spanning multiple releases as it will affect live data. You can use this opportunity to observe first-hand how it's done.